cve_bin_tool/output_engine/__init__.py

```diff
@@ -587,18 +587,18 @@ def output_cves(self, outfile, output_type="console"):
 
     def generate_vex(self, all_cve_data: Dict[ProductInfo, CVEData], filename: str):
         analysis_state = {
-            Remarks.NewFound: "under_review",
-            Remarks.Unexplored: "under_review",
+            Remarks.NewFound: "in_triage",
+            Remarks.Unexplored: "in_triage",
             Remarks.Confirmed: "exploitable",
             Remarks.Mitigated: "not_affected",
             Remarks.Ignored: "not_affected",
         }
         response_state = {
-            Remarks.NewFound: "Outstanding",
-            Remarks.Unexplored: "Not defined",
-            Remarks.Confirmed: "Upgrade required",
-            Remarks.Mitigated: "Resolved",
-            Remarks.Ignored: "No impact",
+            Remarks.NewFound: [],
+            Remarks.Unexplored: [],
+            Remarks.Confirmed: ["update"],
+            Remarks.Mitigated: [],
+            Remarks.Ignored: [],
```

Comment:
A bit sad to 'lose information' here, but the schema only allows a small number of responses, see https://github.com/CycloneDX/specification/blob/master/schema/bom-1.4.schema.json#L1591

Reply:
Hm, it is possible to keep the extra data in a comment field if it's not viable to put it elsewhere?

Reply:
I put it as a prefix in the 'detail' field - though it doesn't impact the processing, as the vex parser never looked at the analysis response anyway, just at the state.